src/CMakeLists.txt

```diff
@@ -72,6 +72,11 @@ set(
 # =======================================================================
 if(LIBHPDF_STATIC)
   add_library(${LIBHPDF_NAME_STATIC} STATIC ${LIBHPDF_SRCS})
+
+  if( CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" )
+      set_target_properties(${LIBHPDF_NAME_STATIC} PROPERTIES COMPILE_FLAGS "-fPIC")
+  endif( CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" )
```

Comment:
This should not be set as default.
The user should set [`CMAKE_POSITION_INDEPENDENT_CODE`](https://cmake.org/cmake/help/latest/variable/CMAKE_POSITION_INDEPENDENT_CODE.html) if they want PIC (or not).